e2e/suite_test.go

```diff
@@ -40,9 +45,25 @@ func TestMain(m *testing.M) {
 		shared.StartGitServer(namespace),
 		shared.InstallFlux("latest"),
 		shared.RunTiltForControllers("ocm-controller"),
-		shared.ForwardPortForAppName("registry", 5000, stopChannelRegistry),
-		shared.ForwardPortForAppName("gitea", 3000, stopChannelGitea),
+		shared.ForwardPortForAppName("registry", registryPort, stopChannelRegistry),
+		shared.ForwardPortForAppName("gitea", gitRepositoryPort, stopChannelGitea),
 	)
+	// testEnv.AfterEachTest(
```

Comment:
Don't forget to add these back :)

Reply:
This actually wasnt there initially, I was trying to experiment. AfterEachTest() needs to go to individual feature if required